spec/controllers/games_controller_spec.rb

```diff
@@ -58,5 +58,16 @@
     end  
   end
 
+  describe "games#update action"  do
+    it "should allow users to join a game as player 2" do
+      user = FactoryGirl.create(:user)
+      sign_in user
+      g = FactoryGirl.create(:game, sith_user_id: user.id) 
+
+      patch :update, id: g.id, jedi_user_id: user.id
+      expect(response).to redirect_to ships_path 
+      user.reload
```

Comment:
If you're not going to do anything with the user, you won't need to reload it here.